examples/existing_vpc/variables.tf

```diff
@@ -32,5 +32,5 @@ variable "existing_resource_group_name" {
 variable "name" {
   description = "The string is used as a prefix for the naming of VPC resources."
   type        = string
-  default     = null
+  default     = "existing-vpc"
```

Comment:
As a note, pre-commit was failing on this line when it was null with the following:

```

TFLint in examples/existing_vpc/:
Failed to prepare rule checking; failed to eval an expression in ../../dynamic_values.tf:7; ../../dynamic_values.tf:7,64-72: Invalid template interpolation value; The expression result is null. Cannot include a null value in a string template.:

Error: Invalid template interpolation value

  on ../../dynamic_values.tf line 7, in module "dynamic_values":
   7:   prefix               = var.prefix != null ? "${var.prefix}-${var.name}" : var.name

with var.name set to null.

The expression result is null. Cannot include a null value in a string template.
```
I added a default string here to get past the pre-commit error, but this may need to have no default/require a value be passed in at all times

Reply:
I'd suggest to make it mandatory